modules/virt-creating-interface-on-nodes.adoc

```diff
@@ -15,7 +15,7 @@ By default, the manifest applies to all nodes in the cluster. To add the interfa
 +
 [source,yaml]
 ----
-apiVersion: nmstate.io/v1alpha1
+apiVersion: nmstate.io/v1beta1
```

Comment:
@phoracek are we still supposed to use `nmstate.io/v1alpha1` ?

Reply:
In 2.5, we still use v1alpha1. We are bumping nmstate CRDs to beta only in the following release.